createManifest.ps1

```diff
@@ -1,3 +1,6 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
```suggestion
# Copyright (c) Microsoft Corporation.
```

Reply:
rights are described in the MIT License

Reply:
Actually, because this violates the Dockerfile standards policy and because this is the copyright header used upstream, I legally can't make that change unless it is also in upstream.

Reply:
that's fine

Reply:
But since there is not copyright header used upstream.  I don't think that rule applies.

Reply:
A look at any `.cs` files upstream show that they do contain the `All rights reserved.` part of it

Reply:
Does Microsoft have an email for legal matters so I can bring this to their attention?